### ISSUE TITLE
Font Awesome Icons in Tabs STILL not working

### ISSUE THREAD

**Issue by ColdstarJoey** [dotnet/maui#24668]
### Description

When azdding the FA OFT font files in the MauiProgram.cs and then using a FontImageSource they do not render still in iOS, any ideas as this was a big part of my navigation?

### Version with bug

8.0.82 SR8.2

### Is this a regression from previous behavior?

Yes, this used to work in Xamarin.Forms

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, I was *not* able test on other platforms

**Comment by mattleibow** [comment]
Do the fonts work anywhere else? Like a label text and in an Image control with the font image source?

**Comment by ColdstarJoey** [comment]
> Do the fonts work anywhere else? Like a label text and in an Image control with the font image source?
Yes as a standard label they show or even if i make a font awesome custom label from online articles, just not in the tabbar or any navigation

**Comment by jaosnz-rep** [comment]
I can repro this issue at iOS platform on the latest 17.12.0 Preview 1.0(8.0.82 & 8.0.80).

**Comment by mattleibow** [comment]
@jaosnz-rep do you have that repro still? Can you attach it here?

**Comment by ColdstarJoey** [comment]
Hi Sorry for the late reply, please see this repo for the same issue

https://github.com/ColdstarJoey/FAMauiShellIssue

**Comment by mattleibow** [comment]
@PureWeen does this sound familiar at all?

**Comment by ColdstarJoey** [comment]
I can confirm as well that this works normally in button / label if I put the FontFamily property as the name defined in MauiProgram.cs ("SolidFont") and the Glyph being the name of the icon ("paper-plane" etc) but not in the Shell obviously

`<Button x:Name="SendEnquiryBtn" Text="Send Enquiry" FontAttributes="Bold" BackgroundColor="Red" TextColor="White">
    <Button.ImageSource>
        <FontImageSource FontFamily="SolidFont" Glyph="paper-plane" Color="White" Size="Small"/>
    </Button.ImageSource>
</Button>`

**Comment by karthikraja-arumugam** [comment]
It appears to be a sample issue. Removing the lines below from the Csproj file resolves it, and the icons work as expected.  

```
<ItemGroup>
	  <MauiFont Remove="Resources\Fonts\Font Awesome 6 Pro-Regular-400.otf" />
	  <MauiFont Remove="Resources\Fonts\Font Awesome 6 Pro-Solid-900.otf" />
	</ItemGroup>

	<ItemGroup>
	  <None Remove="Resources\Fonts\Font Awesome 6 Pro-Regular-400.otf" />
	  <None Remove="Resources\Fonts\Font Awesome 6 Pro-Solid-900.otf" />
	</ItemGroup>
```

https://github.com/ColdstarJoey/FAMauiShellIssue/blob/main/FAMauiShellIssue/FAMauiShellIssue/FAMauiShellIssue.csproj#L58-L68
